protostar/cli/command_names_delayed_provider_test.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from protostar.configuration_file import CommandNamesProvider
+
+from .command_names_delayed_provider import CommandNamesDelayedProvider
+
+
+class CommandNamesProviderDouble(CommandNamesProvider):
```

Comment:
Don't we have three classes with the same name? We cannot do that, it is *very* misleading (for grep users is somewhat hardcore)

Reply:
You could parametrize it and put it in conftest

Reply:
I'd have to pollute global conftest.py, but that fixture would be used only by two components.

Reply:
If it's only two you can resolve it, If it's three I would consider polluting

Reply:
Hmm, I can export the Double (Stub) from the `protostar.configuration_file` component instead of putting it to the global conftest. Maybe in the future. Resolving.